Source/Evolve/WarpXEvolve.cpp

```diff
@@ -394,7 +396,9 @@ WarpX::OneStep_nosub (Real cur_time)
     ExecutePythonCallback("particlescraper");
     ExecutePythonCallback("beforedeposition");
 
+    WARPX_PROFILE_REGION_START("WarpX::nosub::PushParticles");
     PushParticlesandDepose(cur_time);
```

Comment:
Do we directly want to put a `WARPX_PROFILE` inside `PushParticlesandDepose` instead?

Reply:
It could. Up for discussion. I stuck to wrapped section of Evolve and OneStep for now, as you generally want bigger, all-encompassing blocks of code for the FullProfiler analysis and probably matching these regions with the overall flow of the code rather than specific functions seemed like a better deal.

(After reading through the code, I was picturing all the OneStep functions instrumented in a matching way, meaning the profile gives consistent cuts no matter how you run WarpX.)